src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java

```diff
@@ -57,8 +57,6 @@
 
   @CommandLine.Option(
       names = "--post-startup-script",
-      defaultValue =
```

Comment:
Should we instead default to the URL of the one hosted in `terra-workspace-manager`?

Reply:
i did set the default in wsm when creating an instance. Are you worried that the CLI might be talking to an old wsm?

Reply:
No, I was worried there wouldn't be a script. This argument overrides the other one I suppose.

Maybe put somewhere in the README where the canonical location is so folks know what's there and have an idea how to write their own.